### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by suvjunmd**

### CODE REVIEW

**Comment by Unit158** [comment]
First of all, you have way too many commits. You'll need to squash them into a reasonable number before this is merged.

File: FormBuilder.cs [hunk 1]
```diff
@@ -22,118 +22,43 @@ namespace OpenRA.TilesetBuilder
...
-		public bool PaletteFromImage = true;
-		public string PaletteFile = "";
-		public string ImageFile = "";
-		public int TileSize = 24;
-
-		private static int ColorDiff(Color color, Color curr)
-		{
-			return Math.Abs(color.R - curr.R) + Math.Abs(color.G - curr.G) + Math.Abs(color.B - curr.B);
-		}
+		private string _srcfile;
```
Comment (Unit158) [inline FormBuilder.cs]: Omit the explicit private and please use plain lowercase naming. (See the [Coding Standard](https://github.com/OpenRA/OpenRA/wiki/Coding-Standard).)

**Comment by suvjunmd** [comment]
Squashed the commits and removed the private access modifiers

File: FormBuilder.cs [hunk 2]
```diff
@@ -218,6 +156,50 @@ public void Save()
 			}
 		}
 
+		void LoadTerrainDefinitions()
+		{
+			var terrainDefinition = new Dictionary<string, TerrainTypeInfo>();
+			var yaml = MiniYaml.DictFromFile("defaults.yaml");
+			terrainDefinition = yaml["Terrain"].ToDictionary().Values.Select(y => new TerrainTypeInfo(y)).ToDictionary(t => t.Type);
+			var i = 0;
```
Comment (Unit158) [inline FormBuilder.cs]: This should be moved closer to where it is used and be given a better name.

File: FormBuilder.cs [hunk 3]
```diff
@@ -218,6 +156,50 @@ public void Save()
...
+
+				surface1.Icon[i] = icon;
+				terrainType[i] = deftype.Value;
+
+				var terrainTypeButton = new ToolStripButton(deftype.Key, icon, TerrainTypeSelectorClicked);
+				terrainTypeButton.ToolTipText = deftype.Key;
+				terrainTypeButton.DisplayStyle = ToolStripItemDisplayStyle.ImageAndText;
+				terrainTypeButton.Tag = i.ToString();
+				terrainTypeButton.ImageAlign = ContentAlignment.MiddleLeft;
+				i++;
```
Comment (Unit158) [inline FormBuilder.cs]: This should be after the line below it.

File: FormBuilder.cs (hunk 2 above)
Comment (suvjunmd) [inline FormBuilder.cs]: I've came up with this method name based on the first variable. Can you suggest something else  ?

File: FormBuilder.Designer.cs [hunk 4]
```diff
@@ -320,8 +320,9 @@ private void InitializeComponent()
 			this.ClientSize = new System.Drawing.Size(908, 596);
 			this.Controls.Add(this.toolStripContainer1);
 			this.Icon = ((System.Drawing.Icon)(resources.GetObject("$this.Icon")));
-			this.Name = "frmBuilder";
-			this.Text = "Tileset Builder 2";
+			this.Name = "frmBuilder";
+            this.StartPosition = System.Windows.Forms.FormStartPosition.CenterScreen;
+            this.Text = "Tileset Builder 2";
```
Comment (Unit158) [inline FormBuilder.Designer.cs]: tabs please.

File: FormBuilder.cs (hunk 2 above)
Comment (Unit158) [inline FormBuilder.cs]: I meant var i :)

File: FormBuilder.cs (hunk 3 above)
Comment (suvjunmd) [inline FormBuilder.cs]: I don't understand what do you mean

Comment (Unit158) [inline FormBuilder.cs]: i++ should be the final line in the iterator here.

**Comment by suvjunmd** [comment]
Replaced spaces with tabs and moved i variable closer to where it is used

**Comment by Unit158** [comment]
:+1:

**Comment by Mailaender** [comment]
Thanks.

**Pull request merged**